library/std/src/sys/windows/c/README.md

```diff
@@ -0,0 +1,9 @@
+The `windows_sys.rs` file is autogenerated from `bindings.txt` and must not
+be edited manually.
+
+To add bindings, edit `bindings.txt` then regenerate using the following command:
+
+    ./x run generate-windows-sys && ./x fmt library/std
```

Comment:
Maybe the generator should run its output through rustfmt too?

Reply:
I think that would be good but what worried me is if people are using any kind of special arguments to `x`. It would be quite hard to make sure we're invoking `x fmt` the same way whereas it's easy enough for the user to adapt the commands if they have any special requirements. Maybe I'm being overly cautious though.

Also a minor issue is that there's a bit of complexity to ensure we run the right `x fmt` for the platform and the way this worked has changed a few times in the past so we may need to adapt to future changes. That said, it does seem to have settled down now.

Reply:
OK, no big deal either way.